pig-latin/othay-ocolatechay/ocolatechay.md

```diff
@@ -0,0 +1 @@
+Youyay apescray omesay arshmallowsmay offyay ethay allsway andyay utpay ityay inyay youryay othay ocolatechay. astesTay eatgray!
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.